linux_os/guide/system/kernel_build_config/var_kernel_config_module_sig_hash.var

```diff
@@ -0,0 +1,20 @@
+documentation_complete: true
+
+title: 'Hash function for kernel module signing'
+
+description: |-
+    The hash function to use when signing modules during kernel build process.
+
+type: string
+
+operator: equals
+
+interactive: false
+
+options:
+    default: sha256
```

Comment:
```suggestion
    default: sha512
```
is there a specific reason to choose the sha256 here?

Reply:
No special reason, RHEL7 and RHEL8 use `sha256` but RHEL9 uses `sha512`.
We can default to `sha512`, no problem.